Kite SDK/PSPrintSDK/OLProduct.h

```diff
@@ -27,5 +27,7 @@
 -(void)setCoverImageToImageView:(UIImageView *)imageView;
```

Comment:
A bit late to the game on these on setCoverImageToImageView & setProductPhotography:toImageView but I do wonder if they shouldn't be in a private interface internal to OLProduct?